pkg/apis/extensions/v1beta1/types.go

```diff
@@ -70,30 +70,49 @@ type SubresourceReference struct {
 	Subresource string `json:"subresource,omitempty"`
 }
 
-// ResourceConsumption is an object for specifying average resource consumption of a particular resource.
-type ResourceConsumption struct {
-	// Resource specifies either the name of the target resource when present in the spec, or the name of the observed resource when present in the status.
-	Resource v1.ResourceName `json:"resource,omitempty"`
-	// Quantity specifies either the target average consumption of the resource when present in the spec, or the observed average consumption when present in the status.
-	Quantity resource.Quantity `json:"quantity,omitempty"`
+// MetricName is an identifier of a Heapster metric.
+type MetricName string
+
+const (
+	// Heapster metric representing CPU usage of a pod (in milli-cores).
+	MetricCPUUsage MetricName = "cpu-usage"
+	// Heapster metric representing Memory usage of a pod (in bytes).
+	MetricMemoryUsage MetricName = "memory-usage"
+)
+
+// MetricUtilization is an object for specifying the arithmetic mean utilization of a particular metric.
+type MetricUtilization struct {
+	// Name specifies either the identifier of the target metric when present in the spec, or the identifier of the observed metric when present in the status.
+	Name MetricName `json:"name"`
+	// Utilization specifies either the target arithmetic mean utilization of the metric when present in the spec, or the observed arithmetic mean utilization when present in the status.
```

Comment:
It's not clear what "utilization of the metric" means. Isn't it just the arithmetic mean of the metric?